src/Renderer/c3DEngine.js

```diff
@@ -32,6 +32,7 @@ function c3DEngine(scene, positionCamera, viewerDiv, debugMode, gLDebug) {
     this.scene3D.sortObjects = false;
     this.width = this.debug ? viewerDiv.clientWidth * 0.5 : viewerDiv.clientWidth;
     this.height = viewerDiv.clientHeight;
+    this.viewerDiv.size = new THREE.Vector2(this.width, this.height);
```

Comment:
Instead replace `c3DEngine`'s widhth/height property with a size and use it in like this:
```js
this.uniforms.WIN_SCALE = { value: c3DEngine().size };
```